examples/Tools/BandManagement/BandManagement.ino

```diff
@@ -102,13 +104,28 @@ String askUser() {
     newBand = GSM_MODE_GSM850_PCS;
   } else if (c == '6') {
     newBand = GSM_MODE_GSM850_EGSM_DCS_PCS;
+  }else if (c == '7') {
+    newBand = GSM_MODE_UMTS;
+  } else if (c == '8') {
+    newBand = GSM_MODE_GSM850_EGSM_PCS_UMTS;
   } else {
     newBand = "GSM_MODE_UNDEFINED";
   }
+
+  // Set the Radio Access Technology to support the 1800 MHz frequency
+  // in accord with the band select 
+  if (c=='2' || c == '4') {
+    if (band.getRAT() != "1,0") {
```

Comment:
Can the RAT stuff go inside the lib?

Reply:
Do we absolutely need to change the RAT, or can the modem determine the best one based on the frequency set?

Reply:
can i check about the stigmatization of the RAT but i'm quite sure that is required time by time, yes it can go inside the library i'll move immediatly.